### PR TITLE
Add triage info for rancher/rancher images

### PR DESCRIPTION
**Pull request by brendarearden**
Add waiting notes for rancher/rancher images. 

Removed quotes from previous istio image notes for consistency.

### CODE REVIEW

File: docs/_data/cves-v2.6.csv [hunk 1]
```diff
@@ -186,8 +186,8 @@ rancher/istio-1.5-migration:0.1.1,libcurl,alpine,CVE-2021-22901,HIGH,https://avd
...
 rancher/istio-1.5-migration:0.1.1,ssl_client,alpine,CVE-2021-28831,HIGH,https://avd.aquasec.com/nvd/cve-2021-28831,1.31.1-r20,,ignore,Image is required only for a deprecated version of the istio feature. Istio v1.5 should no longer be used and this image should only be included in air gapped registry if necessary.
-rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting," ""Upstream Issue: https://github.com/istio/istio/issues/34179. This CVE does not impact istio's security"""
-rancher/istio-installer:1.10.2-rancher1,github.com/ulikunitz/xz,alpine,CVE-2021-29482,HIGH,https://avd.aquasec.com/nvd/cve-2021-29482,v0.5.8,,waiting," ""Issue: https://github.com/rancher/rancher/issues/33683"""
+rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting,"Upstream Issue: https://github.com/istio/istio/issues/34179. This CVE does not impact istio's security"
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:189]: Should this be in ignore or waiting ? I don't see in that issue that it doesn't impact istio, does it ? If so then: 
```suggestion
rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting,"This CVE does not impact istio's security: https://github.com/istio/istio/issues/34179."

```

File: docs/_data/cves-v2.6.csv [hunk 2]
```diff
@@ -186,8 +186,8 @@ rancher/istio-1.5-migration:0.1.1,libcurl,alpine,CVE-2021-22901,HIGH,https://avd
...
-rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting," ""Upstream Issue: https://github.com/istio/istio/issues/34179. This CVE does not impact istio's security"""
-rancher/istio-installer:1.10.2-rancher1,github.com/ulikunitz/xz,alpine,CVE-2021-29482,HIGH,https://avd.aquasec.com/nvd/cve-2021-29482,v0.5.8,,waiting," ""Issue: https://github.com/rancher/rancher/issues/33683"""
+rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting,"Upstream Issue: https://github.com/istio/istio/issues/34179. This CVE does not impact istio's security"
+rancher/istio-installer:1.10.2-rancher1,github.com/ulikunitz/xz,alpine,CVE-2021-29482,HIGH,https://avd.aquasec.com/nvd/cve-2021-29482,v0.5.8,,waiting,"Issue: https://github.com/rancher/rancher/issues/33683"
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:190]: Waiting should only be used for upstream blockers, just leave state empty. For note put "Tracked in:" or "Issue:"

File: docs/_data/cves-v2.6.csv [hunk 3]
```diff
@@ -695,17 +695,17 @@ rancher/mirrored-istio-galley:1.5.9,sudo,ubuntu,CVE-2021-3156,HIGH,https://avd.a
...
 rancher/mirrored-istio-galley:1.5.9,golang.org/x/crypto,ubuntu,CVE-2020-29652,HIGH,https://avd.aquasec.com/nvd/cve-2020-29652,v0.0.0-20201216223049-8b5274cf687f,,ignore,Image is required only for a deprecated version of the istio feature. Istio v1.5 should no longer be used and this image should only be included in air gapped registry if necessary.
 rancher/mirrored-istio-galley:1.5.9,golang.org/x/crypto,ubuntu,CVE-2020-9283,HIGH,https://avd.aquasec.com/nvd/cve-2020-9283,v0.0.0-20200220183623-bac4c82f6975,,ignore,Image is required only for a deprecated version of the istio feature. Istio v1.5 should no longer be used and this image should only be included in air gapped registry if necessary.
-rancher/mirrored-istio-install-cni:1.10.2,libsystemd0,ubuntu,CVE-2021-33910,HIGH,https://avd.aquasec.com/nvd/cve-2021-33910,237-3ubuntu10.49,,waiting," ""Issue: https://github.com/rancher/rancher/issues/33683"""
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:698]: Same here and below, since 1.10.2 is out then we aren't blocked, and shouldn't be in waiting. Note is fine.

**Review by cbron — CHANGES_REQUESTED** [review]

**Review by rmweir — CHANGES_REQUESTED** [review]
Can approve/merge once @cbron feedback has been addressed.

File: docs/_data/cves-v2.6.csv [hunk 4]
```diff
@@ -187,8 +187,8 @@ rancher/istio-1.5-migration:0.1.1,libcurl,alpine,CVE-2021-22901,HIGH,https://avd
...
 rancher/istio-1.5-migration:0.1.1,ssl_client,alpine,CVE-2021-28831,HIGH,https://avd.aquasec.com/nvd/cve-2021-28831,1.31.1-r20,,ignore,Image is required only for a deprecated version of the istio feature. Istio v1.5 should no longer be used and this image should only be included in air gapped registry if necessary.
-rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting," ""Upstream Issue: https://github.com/istio/istio/issues/34179. This CVE does not impact istio's security"""
-rancher/istio-installer:1.10.2-rancher1,github.com/ulikunitz/xz,alpine,CVE-2021-29482,HIGH,https://avd.aquasec.com/nvd/cve-2021-29482,v0.5.8,,waiting," ""Issue: https://github.com/rancher/rancher/issues/33683"""
+rancher/istio-installer:1.10.2-rancher1,github.com/docker/distribution,alpine,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting,This CVE does not impact istio's security: https://github.com/istio/istio/issues/34179.
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:190]: Still confused why we say it doesn't impact security when the issue has no context

File: docs/_data/cves-v2.6.csv [hunk 5]
```diff
@@ -1868,9 +1868,9 @@ rancher/pipeline-tools:v0.1.16,libxml2,alpine,CVE-2021-3517,HIGH,https://avd.aqu
...
 rancher/rancher-operator:v0.1.4,github.com/gogo/protobuf,alpine,CVE-2021-3121,HIGH,https://avd.aquasec.com/nvd/cve-2021-3121,v1.3.2,,triaged,
 rancher/rancher-operator:v0.1.4,golang.org/x/crypto,alpine,CVE-2020-29652,HIGH,https://avd.aquasec.com/nvd/cve-2020-29652,v0.0.0-20201216223049-8b5274cf687f,,triaged,
-rancher/rancher:master-head,github.com/docker/distribution,suse linux enterprise server,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,triaged,
-rancher/rancher:master-head,github.com/opencontainers/runc,suse linux enterprise server,CVE-2019-16884,HIGH,https://avd.aquasec.com/nvd/cve-2019-16884,v1.0.0-rc8.0.20190930145003-cad42f6e0932,,triaged,
-rancher/rancher:master-head,github.com/opencontainers/runc,suse linux enterprise server,CVE-2019-19921,HIGH,https://avd.aquasec.com/nvd/cve-2019-19921,v1.0.0-rc9.0.20200122160610-2fc03cc11c77,,triaged,
+rancher/rancher:master-head,github.com/docker/distribution,suse linux enterprise server,CVE-2017-11468,HIGH,https://avd.aquasec.com/nvd/cve-2017-11468,v2.7.0-rc.0+incompatible,,waiting,"Waiting on upstream Helm. This CVE does not impact Helm's security.",
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:1871]: This one needs tracking issue so it matches two lines below it.

File: docs/_data/cves-v2.6.csv (hunk 4 above)
Comment (cbron) [reply docs/_data/cves-v2.6.csv:180]: For future reference we talked with them separately and confirmed this.

**Review by cbron — APPROVED** [review]

**Pull request merged**